specification/cognitiveservices/data-plane/TranslatorText/Batch/preview/v1.0/TranslatorBatch.json

```diff
@@ -0,0 +1,1463 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Batch Text Translation API",
+    "version": "v1.0-preview.1"
+  },
+  "host": "westus2.cognitiveservices.azure.com",
+  "basePath": "/translator/text/batch",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/v1.0-preview.1/documents": {
+      "post": {
+        "tags": [
+          "Translation"
+        ],
+        "summary": "Submit a batch document translation request to the translation service",
+        "description": "Submit a batch request to the document translation service.\r\n            \r\nEach request can consists of multiple inputs.\r\nEach input will contains both a source and destination container for source and target language pair.\r\n            \r\nThe prefix and suffix filter (if supplied) will be used to filter the folders.\r\nThe prefix will be applied to the subpath after the container name\r\n            \r\nGlossaries / Translation memory can be supplied and will be applied when the document is being translated.\r\nIf the glossary is invalid or unreachable during translation time.  An error will be indicated in the document status.\r\n            \r\nIf the file with the same name already exists in the destination, it will be overwritten.\r\nTargetUrl for each target language needs to be unique.",
+        "operationId": "Batch Documents Submit",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "body",
+            "name": "body",
+            "description": "request details",
+            "schema": {
+              "$ref": "#/definitions/JobSubmissionBatchRequest"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Successful request and the batch request is created by the service.  The header Operation-Location will indicate a status url with the operation id.",
+            "headers": {
+              "Operation-Location": {
+                "description": "Location of the operation",
+                "type": "string",
+                "format": ""
+              }
+            }
+          },
+          "400": {
```

Comment:
It would make the swagger more readable/easier to follow if you collected all the shared responses in the responses section and referenced them from there...